rfcs/0050-roadmaps.md

```diff
@@ -0,0 +1,320 @@
+- **Roadmap Summary/Description:** A brief description of the roadmap
+
+### Establishing Scope and Themes for a new Roadmap
+
+For each roadmap, a set of **scope** and **themes** should also be defined, in order for TVM's
+community members to quickly learn about a roadmap's key focus areas.
+
+**Establishing Scope**
+
+- How are the tasks tracked in this roadmap grouped together? How can we think about this grouping
+  distinct from those made in other roadmaps? It's okay for there to be overlap with other roadmaps,
+  but the scope defined here should motivate a separate roadmap.
+- Is the proposed roadmap intended to represent a perpetually ongoing set of efforts, or is there an
+  end goal which will close/finalize the roadmap?
+- Does the proposed roadmap have any scope overlaps with any existing roadmaps? If so, please list
+  them.
+
+**Establishing Themes**
+
+- List 4-6 proposed "themes" of the roadmap, intended to convey the purpose of the roadmap and the
+  types of tasks that should be added.
+    - Some examples of themes are `programmability`, `portability`, and `performance`.
+    - Themes are intended to group items within a roadmap. This helps us to understand the scope of
+    the roadmap.
+- For each theme, include a set of definitions specific to the proposed roadmap.
+    - What does this theme mean in the context of this roadmap?
+    - Are there multiple definitions for this theme? For example, `performance` could be interpreted
+    as tuning times, runtime latency, or a number of other definitions.
+- For each theme, include a set of success criteria specific to the proposed roadmap.
+    - What types of metrics would be relevant to this roadmap?
+
+### Adding Items to a Roadmap
+
+A roadmap's maintainers are the primary folks responsible for adding, modifying, and removing items
+for a roadmap. This isn't a hard and fast rule—for example, it may be expedient for other community
+members to triage new roadmap items into a roadmap's backlog. However, the maintainers should be
+considered the "owners" of a roadmap, and generally no rigid process is defined around modifying the
+items in a roadmap.
+
+Roadmaps are defined using [GitHub
+Projects](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-projects)
+and can include GitHub Issues, Pull Requests, and simple note cards. Maintainers should strive to
+place mainly **GitHub Issues** in Roadmaps to make it possible for the community to learn more about
+ongoing work and reduce triage burden.
+
+Each item on a roadmap is intended to track one of these community processes:
+
+- **pre-RFCs** serve as a way to begin discussions on planned work at the earliest stage of
+  maturity. **pre-RFCs** are typically posted in the [TVM Discuss
+  forums](http://discuss.tvm.apache.org) in order to solicit TVM Community feedback. For an example
+  of a **pre-RFC**, see the screenshot of Andrew R's proposal to *Convert RST Docs to Markdown*
```

Comment:
Are we expecting pre-RFCs to be this light weight? Following the same template as the eventual RFC both ensures people are considering the areas in the RFC process as well as starting the process of creating that eventual RFC document.

Reply:
This came up in some of our previous discussions, and we've purposefully left this ambiguous because pre-RFC structures/guidelines are outside the scope of this RFC. 

I do 💯 agree that there should be a set of guidelines for pre-RFCs, and I'd love to discuss this further. Perhaps it can be a topic in the next TVM community meeting! 😸